app/controllers/admin/users_controller.rb

```diff
@@ -57,6 +48,19 @@ def destroy
 
   private
 
+  def create_user
+    User.transaction do
+      @user.save
+      begin
+        @user.create_with_auth0
+      rescue Auth0::Exception => e
+        flash[:alert] = 'There was an error adding the user to Auth0. Please try again.'
+        Rails.logger.warn("Error adding user #{@user.email} to Auth0 during User#create, message: #{e.message}")
```

Comment:
Is this a full-on `logger.error`?

Reply:
(sorry, realise from looking above that this wasn't the subject of the PR)

Reply:
Good question though, are warns picked up by Papertrail?

Reply:
I *think* everything goes. `INFO` from Sidekiq is certainly going, although a [search](https://papertrailapp.com/groups/10314582/events?q=%22Error%20adding%20user%22&focus=1032015685247307794) for `"Error adding user"` returns no results, so we can't see any history of this particular one. It [looks](https://github.com/dxw/DataSubmissionServiceAPI/search?q=%22logger.warn%22&unscoped_q=%22logger.warn%22) like this is the only `logger.warn` in the whole app. [Searching](https://papertrailapp.com/groups/10314582/events?q=WARN) for `WARN` does return plenty but they look like Sidekiq warnings. In any case, I'll argue that this is literally an `Exception`al circumstance so we should take the opportunity to make it an `error`. It's probably entirely moot since it seems never to happen 😄